### PR TITLE
Include VBF to be used in BDT

### PR DESCRIPTION
**Pull request by ClaudioSMV**

### CODE REVIEW

File: plot_macros/plot_roc_space.py
Comment (MatBarria) [inline plot_macros/plot_roc_space.py:1]: line 49: add an option to draw just one of the eras

File: python/xgboost/train.py [hunk 1]
```diff
@@ -80,56 +89,62 @@
...
     ["cos_theta_CS", "cos_theta_CS", r"$cos(\theta_{CS})$", 50, -1, 1],
-    #   Jet variables
+    # # Jet variables
     ["n_jet", "n_jet", r"n jet", 8, 0, 8],
     ["leading_jet_pt", "leading_jet_pt", r"$p_T^{j1}$", 50, 0, 400],
     ["leading_jet_eta", "leading_jet_eta", r"$\eta_{j1}$", 50, -5, 5],
-    ["subleading_jet_pt", "subleading_jet_pt", r"$p_T^{j2}$", 50, 0, 400],
-    ["diJet_mass", "diJet_mass", r"$m_{jj}$ GeV", 50, 0, 400],
-    ["delta_eta_diJet", "delta_eta_diJet", r"$\Delta\eta_{jj}$", 50, -8, 8],
+    ["subleading_jet_pt", "subleading_jet_pt", r"$P_{t}^{j2}$ [GeV]", 50, 0, 400],
```
Comment (MatBarria) [inline python/xgboost/train.py:120]: why P and not p, we are using p in all the other plots I think

File: python/xgboost/train.py [hunk 2]
```diff
@@ -80,56 +89,62 @@
...
-        3.14,
-    ],
-    ["weight_no_lumi", "weight_no_lumi", "weight_no_lumi", 100, -1.0, 1.0],
+    ["z_zeppenfeld", "z_zeppenfeld", r"$Z^{*} Zeppendfeld$", 50, -8, 8],
+    ["min_delta_eta_diMuon_jet", "min_delta_eta_diMuon_jet",
+     r"min$|\Delta\eta_{\mu\mu,j}|$", 50, 0, 8],
+    ["min_delta_phi_diMuon_jet", "min_delta_phi_diMuon_jet",
+     r"min$\Delta\phi_{\mu\mu,j}$", 50, -3.14, 3.14],
 ]
+if channel_US == "ggH":
```
Comment (MatBarria) [inline python/xgboost/train.py:128]: That bins are not use in the code, so you can just put it in the other list and do not add it for each category.

File: plot_macros/plot_roc_space.py
Comment (ClaudioSMV) [reply plot_macros/plot_roc_space.py:1]: Option "--only" added to draw a single era!

File: python/xgboost/train.py (hunk 2 above)
Comment (ClaudioSMV) [reply python/xgboost/train.py:128]: Removed binning info (was unused) and moved variables to avoid having them duplicated

**Review by MatBarria — COMMENTED** [review]
Looks good

**Pull request merged**